### PR TITLE
Add requests-gracedb

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This is a replacement for `ligo-requests`, which can archived once this is ready, I think.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this is ready to go  - the tests fail on windows, but last I checked it's still ok to use `noarch: python`.

File: recipes/requests-gracedb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+test:
+  imports:
+    - requests_gracedb
+  requires:
+    - pytest
+    - pytest-freezegun
+    - pytest-httpserver
+    - pytest-socket
+    - python >=3
```
Comment (chrisburr) [inline recipes/requests-gracedb/meta.yaml]: I guess the package supports Python 2 and 3 but the tests only support Python 3?

(It's not a problem, just really unusual)

Comment (duncanmmacleod) [reply recipes/requests-gracedb/meta.yaml]: No, the tests support python2, but need `mock` for that, and I thought that this was better (read: easier) than adding `- mock  # [py<30]`. Happy to change if requested.

Comment (chrisburr) [reply recipes/requests-gracedb/meta.yaml]: Ah that makes sense! LGTM 👍

**Comment by duncanmmacleod** [comment]
Thanks @chrisburr!

**Pull request merged**